python/cuml/linear_model/mbsgd_regressor.pyx

```diff
@@ -184,42 +194,21 @@ class MBSGDRegressor(Base, RegressorMixin):
                                           convert_dtype=convert_dtype)
         return preds
 
-    def get_params(self, deep=True):
-        """
-        Scikit-learn style function that returns the estimator parameters.
-
-        Parameters
-        -----------
-        deep : boolean (default = True)
-        """
-
-        params = dict()
-        variables = ['loss', 'penalty', 'alpha', 'l1_ratio', 'fit_intercept',
-                     'epochs', 'tol', 'shuffle', 'learning_rate', 'eta0',
-                     'power_t', 'batch_size', 'n_iter_no_change', 'handle']
-        for key in variables:
-            var_value = getattr(self, key, None)
-            params[key] = var_value
-        return params
-
-    def set_params(self, **params):
-        """
-        Sklearn style set parameter state to dictionary of params.
-
-        Parameters
-        -----------
-        params : dict of new params
-        """
+    def get_param_names(self):
+        return super().get_param_names() + \
+            [
+                "loss",
```

Comment:
Just so I'm clear on the pattern - we should always _not_ include handle as a param, right?

Reply:
`handle`, `verbose` and `output_type` will all returned by `Base.get_param_names()`. So you are correct, the pattern is to call `super().get_param_names()` and add only param names for _this class only_ not any base classes.